### DISCUSSION TITLE
Trarget device

### DISCUSSION THREAD

**Discussion by arist0v** [WebThingsIO/android-app#36] [General]
We should target the following devices :

- cellular
- tablet

Should we plan to have comparibility for the following one:

- chrome os(chromebook)
- google tv(could be nice for camera type dashboard)

**Comment by benfrancis** [comment]
I suggest starting with phone + tablet.

Chrome OS can natively use the installable web app and Google TV will need a dedicated 10ft UI.

**Reply by arist0v** [reply]
Yes i agree, i just wanted to be sure we could plan this device in later developpement phase
also just a note that jetpack compose can adapt the gui depending on the screen size, so it could help reduce the work of producing a gui for google tv, but since we will have to work with remote and stuff, i agree that other device than phone and tablet should be added in late release so we could focuse on the most important device first
